legacy-adapter/src/Rsbc.Dmf.LegacyAdapter/Controllers/CasesController.cs

```diff
@@ -412,7 +412,7 @@ public ActionResult GetDocuments([FromRoute] string caseId)
             [FromForm] IFormFile file,
             [FromForm] string priority = "Regular",
             [FromForm] string assign = null,
-            [FromForm] string submittalStatus = null,
+            [FromForm] string submittalStatus = "Accept",
```

Comment:
Please default this to null rather than accept.  Keep in mind that we still have environments that will not be using this feature.